input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md

```diff
@@ -1,8 +1,7 @@
 <p class="request-for-feedback">Input is requested on the appropriateness of allowing Missing Data or Suppressed Data for all elements. Please comment by raising <a href="https://jira.hl7.org/projects/FHIR/issues">HL7 Jira Issues</a>.</p>
 
 ### Profile specific implementation guidance
-- This profile provides a shared base that is common across requests for specific domains. Where a profile is defined for a specific domain that profile **SHALL** be used:
+- This abstract (non-implementable) profile provides a shared base that is common across requests for specific domains. Note the following domain-specific requests, associated profiles, and usage requirements:
   - pathology requests **SHALL** use [AU eRequesting Pathology Request](StructureDefinition-au-erequesting-servicerequest-path.html) profile
   - imaging requests **SHALL** use [AU eRequesting Imaging Request](StructureDefinition-au-erequesting-servicerequest-imag.html) profile 
```

Comment:
I agree with changes to reflect the abstract nature of this profile. I suggest a slight change:

> This abstract profile provides a shared base that is common across requests for specific domains. It is not intended for direct implementation. Where a profile is defined for a specific domain that profile **SHALL** be used:
> - pathology requests **SHALL** use [AU eRequesting Pathology Request](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/master/StructureDefinition-au-erequesting-servicerequest-path.html) profile
> - imaging requests **SHALL** use [AU eRequesting Imaging Request](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/master/StructureDefinition-au-erequesting-servicerequest-imag.html) profile

Suggestions are welcome! @michael-wilson-au, any thoughts?

Reply:
I'm happy with the suggested change from @dbojicic